src/_pytest/outcomes.py

```diff
@@ -80,7 +80,8 @@ def skip(msg="", **kwargs):
     Skip an executing test with the given message.
 
     This function should be called only during testing (setup, call or teardown) or
-    during collection by using the ``allow_module_level`` flag.
+    during collection by using the ``allow_module_level`` flag.  This function can
+    be called in doctest as well.
```

Comment:
```suggestion
    be called in doctests as well.
```

But not sure if it is useful to mention doctests here explicitly..?!

Reply:
I'm OK either way.  I thought to mention it since doctests are special enough for how they are executed but yet very popular.